tests/installation/logs_from_installation_system.pm

```diff
@@ -59,6 +59,7 @@ sub run {
     # We don't change network setup here, so should work
     # We don't parse logs unless it's detect_yast2_failures scenario
     $self->save_upload_y2logs(no_ntwrk_recovery => 1, skip_logs_investigation => !get_var('ASSERT_Y2LOGS'));
+    set_var('DESKTOP', 'gnome') if (get_var('KDE2GNOME'));
```

Comment:
This belongs more into `migration/post_upgrade.pm` than this module, which is only for collecting logs.

Is a new variable necessary? Would `is_sle('<12') && check_var('DESKTOP', 'kde')` be enough?